web/src/connectors/task-annotator-connector/use-split-validation.ts

```diff
@@ -78,24 +81,54 @@ export default function useSplitValidation({
         { enabled: isSplitValidation }
     );
 
-    const userPages: AnnotationsByUserObj[] = useMemo(() => {
-        if (!byUser) {
+    const { data: pages } = useGetPageSummary(
+        { taskId: task?.id, taskType: task?.is_validation },
+        { enabled: Boolean(task) }
+    );
+
+    const { data: latestRevision } = useLatestAnnotationsByUser(
+        {
+            fileId,
+            jobId: job?.id,
+            pageNumbers: pages?.annotated_pages // передать сюда annotated из page sammury
+        },
+        { enabled: isSplitValidation }
+    );
+
+    const latestRevisionByAnnotators: UserRevision[] = useMemo(() => {
+        if (!latestRevision) {
             return [];
         }
-        return byUser[currentPage].filter((userPage) => userPage.user_id !== userId);
-    }, [byUser, currentPage]);
 
-    const taxonLabels = useAnnotationsTaxons(userPages);
+        const result = convertToRevisionByUserArray(latestRevision as unknown as LatestRevisionObj);
```

Comment:
Do we really need to cast types like this? Why can't we use `latestRevision` as is? 

Maybe we need to change return type of `useLatestAnnotationsByUser`? Because it looks like a mistake here, object can't have `{[key: number]: value}` because keys are always strings

Reply:
Also maybe we can move `const result = convertToRevisionByUserArray(latestRevision as unknown as LatestRevisionObj);` to separate variable to reuse it in `latestRevisionByAnnotators` and `latestRevisionByCurrentUser`? For me it looks like the same code

Reply:
fixed https://github.com/epam/badgerdoc/pull/584/commits/79c03c121c6c6cbd531b96f01677773062a1bb65